src/gpuarray_buffer_cuda.c

```diff
@@ -147,13 +148,49 @@ static int setup_lib(void) {
         return error_set(global_err, GA_IMPL_ERROR, "cuDriverGetVersion failed");
       major = tmp / 1000;
       minor = (tmp / 10) % 10;
+      #if defined(WIN32) || defined(_WIN32) || defined(WIN64) || defined(_WIN64) || defined(__APPLE__)
+      /* We will dynamically search the right CUDA version only on Windows and Macintosh systems,
+      and only if user has not explicitely specified GPUARRAY_CUDA_VERSION. */
+      search_version = 1;
+      #endif
     } else {
       major = ver[0] - '0';
       minor = ver[1] - '0';
     }
+    /* NB: next line will cause problems if a CUDA 10.0 (or 9.11) is released in the future. */
     if (major > 9 || major < 0 || minor > 9 || minor < 0)
       return error_fmt(global_err, GA_VALUE_ERROR, "Invalid cuda version: %d.%d", major, minor);
-    res = load_libnvrtc(major, minor, global_err);
+    if (!search_version) {
+      res = load_libnvrtc(major, minor, global_err);
+    } else {
+      /* First case in next array is reserved to eventually receive the version returned by cuDriverGetVersion(). */
+      int versions[] = {-1, 80, 75};
```

Comment:
We still support 7.0.

Also, since cuda 10 is not so far away, make this a list of pair of ints instead to explicitly list the majors/minors that we support.